### PR TITLE
except ImportError and reraise with updated message

### PR DESCRIPTION
**Pull request by grqz**

### CODE REVIEW

File: plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-from yt_dlp import YoutubeDL
 from yt_dlp.networking.common import Request
 from yt_dlp.networking.exceptions import RequestError, UnsupportedRequest
 
-from yt_dlp_plugins.extractor.getpot import GetPOTProvider, register_preference, register_provider
+try:
+    from yt_dlp_plugins.extractor.getpot import GetPOTProvider, register_preference, register_provider
+except ImportError as e:
+    e.msg += '\nyt-dlp-get-pot is missing! See https://github.com/coletdjnz/yt-dlp-get-pot?tab=readme-ov-file#installing.'
+    raise e from e
```
Comment (grqz) [inline plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py:15]: do we need `from e`?

Comment (Brainicism) [reply plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py:15]: I don't think its important here, the exception is clear regardless

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py (hunk 1 above)
Comment (grqz) [reply plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py:15]: do you want to remove the try block?

Comment (Brainicism) [reply plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py:15]: Let's keep it to be explicit, but i dont think the `from e` is necessary

Comment (grqz) [reply plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py:15]:
```suggestion
    raise e
```

File: plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py [hunk 2]
```diff
@@ -2,12 +2,19 @@
...
+    from yt_dlp import YoutubeDL
 from yt_dlp.networking.exceptions import RequestError, UnsupportedRequest
 from yt_dlp.utils import Popen, classproperty
 
-from yt_dlp_plugins.extractor.getpot import GetPOTProvider, register_preference, register_provider
+try:
+    from yt_dlp_plugins.extractor.getpot import GetPOTProvider, register_preference, register_provider
+except ImportError as e:
+    e.msg += '\nyt-dlp-get-pot is missing! See https://github.com/coletdjnz/yt-dlp-get-pot?tab=readme-ov-file#installing.'
+    raise e from e
```
Comment (grqz) [inline plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py:16]:
```suggestion
    raise e
```

**Review by Brainicism — APPROVED** [review]

**Pull request merged**